packages/client-core/i18n/en/common.json

```diff
@@ -76,6 +76,7 @@
     "tick": "Tick",
     "state": "State",
     "systems": "Systems",
+    "uuid": "Avg Duration: ",
```

Comment:
doesn't look like the right string for this but I could be wrong.

Reply:
Ya, that's not very clear. I renamed it